how-to-docs/amazon-resources.md

```diff
@@ -0,0 +1,55 @@
+# How to get Amazon Resources in the UK Cluster
+
+## Workflow
+1. Pull stacks-hod-platform repo
+2. Create/request credentials and secrets
+3. Configure templates in feature branch
+4. Create a pull request, and an _Amazon Resource Request_ in the hub
```

Comment:
Could you confirm the ARR in the Hub covers the following:
- Field to link PR
- Business Case ? (- Check with Rick if this is required)
- Team / Cost Code ? (- Check with Rick, these resources would be specific to a project so need to understand how it's charged to them)